modules/oadp-restic-issues.adoc

```diff
@@ -11,7 +11,7 @@ You might encounter these issues when you back up applications with Restic.
 [id="restic-permission-error-nfs-root-squash-enabled_{context}"]
 == Restic permission error for NFS data volumes with root_squash enabled
 
-The `Restic` pod log displays the error message, `controller=pod-volume-backup error="fork/exec/usr/bin/restic: permission denied"`.
+The `Restic` pod log displays the error message: `controller=pod-volume-backup error="fork/exec/usr/bin/restic: permission denied"
```

Comment:
Technically this should still have a period at the end of the sentence. But more importantly, it has a mismatching backtick

```suggestion
The `Restic` pod log displays the error message: `controller=pod-volume-backup error="fork/exec/usr/bin/restic: permission denied"`.
```

Reply:
Fixed.

Reply:
Nice catch -- not in my PR, so I'm glad you read the whole file.